modules/core/src/main/java/org/apache/synapse/mediators/builtin/CallMediator.java

```diff
@@ -134,6 +140,10 @@ private boolean handleBlockingCall(MessageContext synInCtx) {
         }
 
         MessageContext resultMsgCtx = null;
+        //If initClientOption is False then set it with blockingMsgsender
+        if (!initClientOptions) {
+            blockingMsgSender.setInitClientOptions(false);
```

Comment:
Can we write this as `blockingMsgSender.setInitClientOptions(initClientOptions)`?

Reply:
Done